docs/en/sql-reference/functions/hash-functions.md

```diff
@@ -1949,3 +1949,39 @@ SELECT sqidDecode('gXHfJ1C6dN');
 │ [1,2,3,4,5]              │
 └──────────────────────────┘
 ```
+
+## keccak256
+
+Calculates Keccak-256 hash string and returns the resulting set of bytes as [FixedString](../data-types/fixedstring.md).
+
+**Syntax**
+
+```sql
+keccak256('s')
+```
+
+This cryptographic hash-function is used a lot in [EVM-based blockchains](https://ethereum.github.io/yellowpaper/paper.pdf). 
+
+**Arguments**
+
+- s - input string for Keccak-256 hash calculation. [String](../data-types/string.md).
+
+**Return value**
+
+- Keccak-256 hash as a byte array with type FixedString(32). [FixedString](../data-types/fixedstring.md).
+
+**Example**
+
+Use function [hex](../functions/encoding-functions.md/#hex) to represent the result as a hex-encoded string.
+
+Query:
+```sql
+select hex(keccak256_go('hello'))
```

Comment:
Why `_go` here?

Reply:
I was testing that the output was the same than my keccak256 UDF written in Golang.
I'm fixing it.